### PR TITLE
[Android] Removed deprecated calls

### PR DESCRIPTION
**Pull request by turtledreams**

### CODE REVIEW

File: android/next.md [hunk 1]
```diff
@@ -226,13 +222,13 @@ CountlyNative.initNative(getApplicationContext());</code></pre>
...
 <h3>Automatic symbol file upload</h3>
-<pre><code class="java">apply plugin: ly.count.android.plugins.UploadSymbolsPlugin 
+<pre><code class="java">apply plugin: ly.count.android.plugins.UploadSymbolsPlugin
 
 buildscript {
     repositories {
         mavenCentral()
     }
-    // for LATEST_VERSION check https://search.maven.org/artifact/ly.count.android/sdk
+    // for LATEST_VERSION check <https://search.maven.org/artifact/ly.count.android/sdk>
```
Comment (turtledreams) [inline android/next.md:231]: zendesk change

File: android/next.md [hunk 2]
```diff
@@ -530,22 +526,18 @@ config.setAutomaticViewSegmentation(automaticViewSegmentation);</code></pre>
...
 </p>
-<pre><code class="java">Countly.sharedInstance().changeDeviceIdWithoutMerge(DeviceId.Type.OPEN_UDID, null)</code></pre>
+<pre><code class="java">Countly.sharedInstance().deviceId().changeWithoutMerge("new device ID")</code></pre>
 <p>
   Doing it this way, will not merge the previously acquired data with the new id.
 </p>
-<p>
-  You can also set Advertising ID as your device ID generation strategy or even
-  supply your own string with <code>DeviceId.Type.DEVELOPER_SPECIFIED</code> type.
-</p>
```
Comment (turtledreams) [inline android/next.md:548]: erasing this is fine?

File: android/next.md [hunk 3]
```diff
@@ -1115,7 +1107,7 @@ Countly.sharedInstance().init(appC, COUNTLY_SERVER_URL, COUNTLY_APP_KEY);</code>
 </p>
 <h2>Manual Remote Config</h2>
 <p>
-  <span style="font-weight: 400;">There are three ways for manually requesting a remote config update: * Manually updating everything * Manually updating specific keys * Manually updating everything except specific keys.</span>
+  <span style="font-weight: 400;">There are three ways for manually requesting a remote config update: *Manually updating everything* Manually updating specific keys * Manually updating everything except specific keys.</span>
```
Comment (turtledreams) [inline android/next.md:1110]: zendesk change

File: android/next.md [hunk 4]
```diff
@@ -77,15 +77,11 @@
   <span style="font-weight: 400;">You may specify the device ID by yourself if you have one (it has to be unique for each device). It may be an email or some other internal ID used by your other systems.</span>
 </p>
 <pre><code class="java">CountlyConfig config = (new CountlyConfig(appC, COUNTLY_APP_KEY, COUNTLY_SERVER_URL));<br>config.setDeviceId("YOUR_DEVICE_ID");<br>Countly.sharedInstance().init(config);</code></pre>
-<p>
-  <span style="font-weight: 400;">You may rely on the Google Advertising ID for device ID generation.</span>
-</p>
-<pre><code class="java">CountlyConfig config = (new CountlyConfig(appC, COUNTLY_APP_KEY, COUNTLY_SERVER_URL));<br>config.setIdMode(DeviceId.Type.ADVERTISING_ID);<br>Countly.sharedInstance().init(config);</code></pre>
```
Comment (turtledreams) [inline android/next.md:83]: removed adevertesing ID

File: android/next.md [hunk 5]
```diff
@@ -77,15 +77,11 @@
...
 <p>
   <span style="font-weight: 400;">In regard to the Google Advertising ID, please ensure you have Google Play services 4.0+ included in your project. Also, note that the Advertising ID silently falls back to OpenUDID in case, it fails to get the Advertising ID when Google Play services are not available on a device.</span>
 </p>
```
Comment (turtledreams) [inline android/next.md:82]: should erase?

File: android/next.md [hunk 6]
```diff
@@ -994,7 +986,7 @@ channel.setSound(soundUri, audioAttributes);</code></pre>
   <span style="font-weight: 400;">You may also completely disable the push notification handling made by the Countly SDK. To do so, just add <code>true</code></span><span style="font-weight: 400;">&nbsp;to the end of your <code>initMessaging()</code></span><span style="font-weight: 400;">call:</span>
 </p>
 <pre><code class="java">Countly.sharedInstance()
-    .init(this, "YOUR_SERVER", "APP_KEY", null, DeviceId.Type.ADVERTISING_ID)
+    .init(this, "YOUR_SERVER", "APP_KEY", null, DeviceIdType.ADVERTISING_ID)
```
Comment (turtledreams) [inline android/next.md:989]: should erase?

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**